exercises/high-scores/canonical-data.json

```diff
@@ -38,52 +38,44 @@
       "description": "Top 3 scores",
       "cases": [
         {
-          "description": "Personal top",
-          "property": "personalTop",
+          "description": "Personal top three from a list of scores",
```

Comment:
Why does this description have to make a point about the top three when I don't see any of the following cases making the same distinction?

Reply:
Yeah. To explain what the point of the method is.